skiko/src/commonTest/kotlin/org/jetbrains/skia/ManagedStringTest.kt

```diff
@@ -23,6 +23,12 @@ class ManagedStringTest {
         assertEquals("你好", s3)
     }
 
+    @Test
+    fun emptyStringTest() {
```

Comment:
What do you think about adding a few more simple tests with intiially empty strings:

`s.insert(0, "")`, `s.append("")`.

Also I'm wondering what is going to happen if we call insert with an index out of bounds. Also `remove` with an index out of bounds.